x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ssl/DefaultJDKTrustConfig.java

```diff
@@ -78,11 +80,11 @@ public int hashCode() {
      * @param trustConfig the trust configuration to merge with
      * @return a {@link TrustConfig} that represents a combination of both trust configurations
      */
-    static TrustConfig merge(TrustConfig trustConfig) {
+    static TrustConfig merge(TrustConfig trustConfig, SecureString trustStorePassword) {
```

Comment:
can you add the parameter to the javadocs and include that this password is for the default jdk store?